.github/workflows/master.yml

```diff
@@ -27,35 +42,35 @@ jobs:
           BUILD_ARGS: /p:AdditionalDefineConstants=SECP256K1_VERIFY
           Framework: netcoreapp3.1
   dotnetcore31standard20:
-    name: '3.1 with netstandard2.0'
+    name: '6.0 with netstandard2.0'
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: '3.1.x'
+          dotnet-version: '6.0.x'
       - run: ./Build/CI/tests.sh
         env:
           BUILD_ARGS: /p:TargetFrameworkOverride=netstandard2.0
-          Framework: netcoreapp3.1
+          Framework: net6.0
   dotnetcore31macod:
-    name: '3.1 on Mac-OS'
+    name: '6.0 on Mac-OS'
```

Comment:
@NicolasDorier pls note with this change, name of the job doesn't match anymore with the job's caption, should be dotnet6macos

Reply:
fixed